CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @NomicFoundation/Slang @NomicFoundation/SlangExtended
```

Comment:
GitHub is complaining that these two teams not being publicly available or not having valid permissions, but they are (AFAICT). Not sure what's happening here.

But also, there is an existing `CODEOWNERS` file inside `.github`, which I think will take precedence over this one. At least [that's what the documentation says](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location).

Reply:
> GitHub is complaining that these two teams not being publicly available or not having valid permissions

Turns out they were needed to be added to the repo explicitly first. Did that now.

> But also, there is an existing CODEOWNERS file inside .github, which I think will take precedence over this one. 

good catch. fixed!